### DISCUSSION TITLE
Best command

### DISCUSSION THREAD

**Discussion by wh1ter0ze** [hktalent/scan4all#5] [Q&A] [answered]
Hi thanks for the tool can u tell me what is the best command to use this tool effectively against a list of hosts

**Comment by hktalent** [comment, marked as answer]
git clone github.com/hktalent/scan4all
cd scan4all
go build -o scan4all main.go
./scan4all -list yourHostLists.txt -o out.json -json

**Reply by hktalent** [reply]
https://github.com/hktalent/scan4all/blob/main/static/running.md

**Reply by wh1ter0ze** [reply]
Thanks
